### PR TITLE
very first commit for Turkish language

### PR DESCRIPTION
**Pull request by ismailunal**
I'm planning to add more. Thanks for this repository. I've kept some of the english terms in place in parentheses in order to reduce confusion. This will be alright when we get more people are around to contribute.

### CODE REVIEW

**Comment by doomspork** [comment]
@ismailunal thank you so much!  This is super awesome.

**Comment by doomspork** [comment]
@hasanozgan do you have the time to look this over?

**Comment by doomspork** [comment]
@ismailunal it appears @hasanozgan may be busy, would you like me to merge these and then you can open PRs for additional version?

**Comment by ismailunal** [comment]
@doomspork it's up to you ;). I'll keep updating / creating more in the future.

**Comment by doomspork** [comment]
@ismailunal if you could fix the conflict with `index.md` I'd love to merge these additions and fire off a tweet about it 😁

**Comment by doomspork** [comment]
Howdy @ismailunal!  Any updates here?  Thanks again for your hard work 😀

**Comment by ismailunal** [comment]
@doomspork  Hi Sean, I can see there is conflicting file but I cannot see any conflicts. How to resolve these could you please suggest something?

File: index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ lang: en
 
 Lessons about the Elixir programming language, inspired by Twitter's [Scala School](http://twitter.github.io/scala_school/).
 
-Available in [Việt ngữ][vi], [汉语][cn], [Español][es], [Slovenčina][sk], [日本語][jp], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Melayu][my], [Українською][uk], [한국어][ko], [L'italiano][it], [Deutsch](de), [বাংলা](bn) and other.
+Available in [Việt ngữ][vi], [汉语][cn], [Español][es], [Slovenčina][sk], [日本語][jp], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Melayu][my], [Українською][uk], [한국어][ko], [L'italiano][it], [Deutsch](de), [বাংলা](bn), [Türkçe](tr) and other.
```
Comment (doomspork) [inline index.md]: It looks like the conflict is on this line.  I believe it should be something akin to this:

```markdown
Available in [Việt ngữ][vi], [汉语][cn], [Español][es], [Slovenčina][sk], [日本語][jp], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Indonesia][id], [Bahasa Melayu][my], [Українською][uk], [한국어][ko], [L'italiano][it], [Deutsch](de), [বাংলা](bn), [Türkçe](tr) and others.
```

You can try to resolve the conflict by rebasing your branch on origin and fixing any conflicts, to do that try:

```elixir
cd elixir-school
git remote add origin git@github.com:doomspork/elixir-school.git
git fetch --all
git rebase origin/master
```

The above commands should notify you of a conflict in the rebase.  Once you fix `en/index.md` you'll need to do:

```elixir
git add en/index.md
git rebase --continue
```

I hope that helps!

**Comment by doomspork** [comment]
@ismailunal howdy!  Would you like to sync up with me in Elixir Slack to work through your PR or would you prefer to close it for now?

**Comment by ismailunal** [comment]
Hey @doomspork. Sorry I have been away for a while. I'll look into this asap and merge more in the future.

**Comment by doomspork** [comment]
No worries @ismailunal, glad to hear from you 😁 Hope all is well with you and yours!

Look forward to your future updates 👍

File: en/index.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
 ---
+layout: page
```
Comment (doomspork) [inline en/index.md]: I think we want to remove `layout: page` here and change `lang: en` below to `redirect_from: /` like it was before.  This is a hiccup as a result of the big refactor 👍

File: tr/lessons/basics/collections.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
+---
+version: 1.0.0
```
Comment (doomspork) [inline tr/lessons/basics/collections.md]: The metadata for lessons has been _greatly_ simplified :tada:, we can update this to just:

```markdown
---
version: 0.9.0
title: Collections
---
```

File: tr/lessons/basics/enum.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
+---
+version: 1.0.0
```
Comment (doomspork) [inline tr/lessons/basics/enum.md]: This metadata can be simplified too 👍

**Comment by ismailunal** [comment]
done ;)

**Comment by doomspork** [comment]
Thank you @ismailunal!!  What is your Twitter handle? 😁

**Pull request merged**